### PR TITLE
test commit for wpt syncing

### PR DESCRIPTION
**Pull request by jdm**
This is just a test of the automated syncing service.

---
This change is [Reviewable](https://reviewable.io/reviews/servo/servo/19921)

### CODE REVIEW

**Comment by asajeffrey** [comment]
Does this test need a kind soul to r+ it? If so, r=me FWIW.

File: tests/wpt/web-platform-tests/2dcontext/building-paths/canvas_complexshapes_arcto_001-ref.htm [hunk 1]
```diff
@@ -4,6 +4,7 @@
         <title>HTML5 Canvas Test:  arcTo() adds to subpath if same point</title>
         <link rel="author" title="Microsoft" href="http://www.microsoft.com" />
     </head>
+    lol
```
Comment (KiChjang) [inline tests/wpt/web-platform-tests/2dcontext/building-paths/canvas_complexshapes_arcto_001-ref.htm]: lol?

Comment (jdm) [reply tests/wpt/web-platform-tests/2dcontext/building-paths/canvas_complexshapes_arcto_001-ref.htm]: It's going to be reverted next time I do a real sync from upstream.

**Pull request merged**